src/com/paynemiller/ch3/SimpleInterestComputer.java

```diff
@@ -0,0 +1,38 @@
+package com.paynemiller.ch3;
+
+import java.util.Scanner;
+
+public class SimpleInterestComputer {
+    Scanner keyboard = new Scanner(System.in);
+    double principalAmount;
+    double rateOfInterest;
+    int numYears;
+    double simpleInterest;
+
+    public void promptForInput(){
+        System.out.print("Enter the principal: ");
+        principalAmount = keyboard.nextDouble();
+        System.out.print("Enter the rate of interest: ");
+        rateOfInterest = keyboard.nextDouble();
+        System.out.print("Enter the number of years: ");
+        numYears = keyboard.nextInt();
+    }
+
+    public void calculateSimpleInterest(){
+        rateOfInterest = rateOfInterest / 100;
+        simpleInterest = principalAmount * (1 + (rateOfInterest * numYears));
+        simpleInterest = Math.round(simpleInterest * 100.0) / 100.0; // rounds to the next decimal
+    }
+
+    public void showSimpleInterest(){
+        System.out.print("After " + numYears + " years at " + rateOfInterest * 100 + "%, the investment will be worth $" + simpleInterest);
+    }
+
+    public void output(){
```

Comment:
Is there a more descriptive name besides `output()`? What does this method do?

Reply:
Changed name of the method.

https://github.com/paynemiller92/programming-practice-erick/pull/12/commits/cc1af3f74b229c1f93959aad8523c7da984a7078